src/main/java/data_access/ApiDataAccessObject.java

```diff
@@ -7,10 +7,10 @@
 import org.json.JSONArray;
 import org.json.JSONObject;
 
-import entity.Card;
```

Comment:
What part of CA is the "data_type" folder? Why have these been removed from Entities?

Reply:
For the same reason GameInfo was removed form entities; these classes hold info that are never changed during program run, so they are not exactly entities.

I moved them because it looked like the API DAO was importing from the entities package, which would violate CA.